OrbitBase/ProfilingTest.cpp

```diff
@@ -27,3 +28,18 @@ TEST(Profiling, ThreadId) {
   EXPECT_TRUE(worker_tid != current_tid);
 }
 #endif
+
+TEST(Profiling, GetSetThreadNames) {
+  const std::string kThreadName = "ProfilingTest";
+  SetThreadName(kThreadName);
+  std::string thread_name = GetThreadName(GetCurrentThreadId());
+  EXPECT_EQ(kThreadName, thread_name);
+
+  // On Linux, the maximum length for a thread name is 16 characters including '\0'.
```

Comment:
Do we want to test this? Is anything bad going to happen to Orbit if linux start limiting thread name with 32 characters?